components/LocationComparisonSection/LocationComparisonBanner.tsx

```diff
@@ -1,20 +1,28 @@
-import React, { CSSProperties, FunctionComponent, useState } from 'react';
+import React, { CSSProperties, FunctionComponent, useEffect, useState } from 'react';
 import { SpotlightLocation } from '../../utils';
 import { Button } from '../Button';
 import { ButtonBanner } from '../ButtonBanner';
 import { LocationSelectionBanner } from '../LocationSelectionBanner';
 import { SpotlightBanner } from '../SpotlightBanner';
 import { TagList, TagListItem } from '../Tags';
+import { Alert } from '../Alert';
+import { Icon } from '../Icon';
 
 interface ComparisonWrapperProps {
   countryName: string;
   countryCode: string;
   onCompare?: (locations: SpotlightLocation[]) => void;
+  queryLocation?: SpotlightLocation[];
```

Comment:
Just name it location... doesn't need to be from a query only. Or is it?

Reply:
Also, since it's taking in an array, plural naming applies.

Reply:
Its actually a prop of `LocationComparisonSection`; Updated it to `locations`